### PR TITLE
feat: add page breadcrumbs

### PR DESCRIPTION
**Pull request by elysee15**
Capture d’écran 2021-09-17 à 00 59 41
Capture d’écran 2021-09-17 à 00 59 03

### CODE REVIEW

File: web/gds-admin-ui/src/pages/app/dashboard/index.js [hunk 1]
```diff
@@ -19,12 +20,15 @@ const ProjectDashboardPage = (): React$Element<React$FragmentType> => {
...
+        dispatch(fecthRegistrationsReviews())
     }, [dispatch])
 
     return (
         <React.Fragment>
             <PageTitle
-                breadCrumbItems={[]}
+                breadCrumbItems={[
+                    { label: 'Summary', path: '/dashboard', active: true }
+                ]}
```
Comment (bbengfort) [inline web/gds-admin-ui/src/pages/app/dashboard/index.js:31]: Very handy!

File: web/gds-admin-ui/src/redux/dashboard/actions.js [hunk 2]
```diff
@@ -46,9 +46,28 @@ const fetchVaspsApiResponseError = (actionType: string, data: any): Action => ({
...
+    }
+})
+
+const fetchRegistrationsReviewsError = (actionType: string, error: any): Action => ({
+    type: FetchRegistrationsReviewsActionTypes.API_RESPONSE_ERROR,
+    payload: {
+        actionType, error
+    }
+})
+
```
Comment (bbengfort) [inline web/gds-admin-ui/src/redux/dashboard/actions.js:70]: I noticed this in the previous PR also - is this just to prepare for fetching data from the API?

**Review by bbengfort — APPROVED** [review]
Excellent!

File: web/gds-admin-ui/src/redux/dashboard/actions.js (hunk 2 above)
Comment (elysee15) [reply web/gds-admin-ui/src/redux/dashboard/actions.js:70]: yes I do it like to make things easier with backend communication

File: web/gds-admin-ui/src/pages/app/dashboard/TasksChart.js [hunk 3]
```diff
@@ -173,8 +173,8 @@ const TasksChart = (): React$Element<any> => {
                 <small>
                     ⋆ click on the elements of the legend to filter accordingly
                 </small>
-            </Card.Body>
-        </Card>
+            </Card.Body >
+        </Card >
```
Comment (bbengfort) [inline web/gds-admin-ui/src/pages/app/dashboard/TasksChart.js:177]: @elysee15 typo?

**Pull request merged**